cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -3204,37 +3204,37 @@ ir.cpp:
 #  561|     r561_2(E)        = Load[e]                : &:r561_1, ~m?
 #  561|     r561_3(int)      = Convert                : r561_2
 #  561|     v561_4(void)     = Switch                 : r561_3
-#-----|   Case[0] -> Block 4
-#-----|   Case[1] -> Block 2
-#-----|   Default -> Block 3
+#-----|   Case[0] -> Block 2
+#-----|   Case[1] -> Block 3
+#-----|   Default -> Block 4
 
 #  560|   Block 1
 #  560|     r560_6(glval<int>) = VariableAddress[#return] : 
 #  560|     v560_7(void)       = ReturnValue              : &:r560_6, ~m?
 #  560|     v560_8(void)       = AliasedUse               : ~m?
 #  560|     v560_9(void)       = ExitFunction             : 
```

Comment:
This also looks a bit strange - I think the common return block is always Block 1 if there are multiple return statements (or a void return statement in a conditional and a return by fallthrough), but the return instructions will be tacked onto another block if there is only one return statement.